docs/UserGuide.md

```diff
@@ -513,32 +727,104 @@ Format: `help`
+----------------------------------------------------------------------
+	 To view all plans, please enter:
+	 plan /list
+	 This will print all the existing plans.
+----------------------------------------------------------------------
+	 To add a plan, please enter: 
+	 plan /new <plan name> /workouts <workout index(s) separated by ','>
+	 Example: 
+	 The workout list upon entering workout /list, contains the
+ 	 following workouts:
+	 1. push up (10 reps)
+	 2. sit up (10 reps)
+	 3. pull up (10 reps)
+	 To create a new plan, enter the following command:
+	 plan /new Grow My Muscles /workouts 1, 2
+	 A new plan named Grow My Muscles with workout index 1 and 2
+	 will be created and added to the application's list of plans.
+----------------------------------------------------------------------
+	 To delete a plan, please enter: 
+	 plan /delete <index of plan>
+	 Example: 
+	 plan /delete 1
+	 This will delete the plan with index 1 if exists.
+----------------------------------------------------------------------
+----------------------------------------------------------------------
+	 To view the schedule, please enter:
+	 schedule /list
+	 This will print the schedule.
+----------------------------------------------------------------------
+	 To update schedule, please enter: 
+	 schedule /update <day index [1-7]> <plan index in the list>
+	 Example: 
+	 The plan list upon entering plan /list, contains the
+ 	 following plans:
+	 1. arms
+	 2. stronger arms
+	 3. Grow My Muscles
+	 To update the plan schedule for Monday with plan 1, enter the
+ 	 following command:
+	 schedule /update 1 1
+	 This will update the schedule of Monday with a plan named arms.
+----------------------------------------------------------------------
+	 To clear schedule, please enter: 
+	 schedule /clear <day index [1-7]>
+	 This will clear the schedule for the given day.
+	 schedule /clearall
+	 This will clear the schedule for everyday.
+----------------------------------------------------------------------
+----------------------------------------------------------------------
+	 To search for exercise(s), please enter: 
+	 search /exercise <exercise keyword>
+	 Example: 
+	 search /exercise up
+	 This will show the exercise(s) containing keyword up if exist.
+----------------------------------------------------------------------
+	 To search for workout(s), please enter: 
+	 search /workout <exercise keyword or number of reps>
+	 Example: 
+	 search /workout up
+	 This will show the workout(s) containing exercise with keyword
+	 up if exist.
+	 search /workout 15
+	 This will show the workout(s) with reps = 15 if exist.
+----------------------------------------------------------------------
+	 To search for plan(s), please enter: 
+	 search /plan <plan keyword>
+	 Example: 
+	 search /plan grow
+	 This will show the plan(s) containing keyword grow if exist.
+----------------------------------------------------------------------
```

Comment:
is search /all inside?

Reply:
Just added that, thank you for finding it!